Assets/LeapMotion/Core/Scripts/Hands/RiggedFinger.cs

```diff
@@ -73,20 +72,7 @@ public class RiggedFinger : FingerModel {
               // the standard "test" edit-time hand model from the TestHandFactory.
               var boneTipPos = GetJointPosition(i + 1);
               var boneVec = boneTipPos - boneRootPos;
-
-              // If the rigged hand is scaled (due to a scaled rig), we'll need to divide
```

Comment:
mm, this code seems like it is a UnityModules-but-not-Sim fix that we should keep and not remove?